### PR TITLE
Add section on how to use this driver with arduino code

### PR DESCRIPTION
**Pull request by marcmerlin**

### CODE REVIEW

**Comment by marcmerlin** [comment]
@hzeller  this replaces https://github.com/hzeller/rpi-rgb-led-matrix/pull/962

File: README.md [hunk 1]
```diff
@@ -140,6 +140,12 @@ network protocols, such as the
 [PixelPusher implementation](https://github.com/hzeller/rpi-matrix-pixelpusher)
 (common in light art installations).
 
+If you have arduino code that uses RGBPanels, you'll likely hit limits around 128x64
```
Comment (hzeller) [inline README.md]: Keep it concise like the other entries.

e.g.

[FastLED/GFX Bridge ](http://marc.merlins.org/perso/arduino/post_2020-01-01_Running-FastLED_-Adafruit_GFX_-and-LEDMatrix-code-on-High-Resolution-RGBPanels-with-a-Raspberry-Pi.html) for Arduino code (porting Arduino code for these to Raspberry Pi).

Comment (marcmerlin) [reply README.md]: Ok, you got me, terse is not my middle name :)
How about this new version?

**Comment by hzeller** [comment]
Sounds good. Do a rebase though, as that section changed over the day.

**Comment by marcmerlin** [comment]
Sure thing, merged against master TOT

**Comment by hzeller** [comment]
thanks, merged.

**Pull request merged**